sources/mysql/mysqldump.go

```diff
@@ -94,6 +94,10 @@ func processMySQLDump(conv *internal.Conv, r *internal.Reader) error {
 // files containing tens or hundreds of GB of data.
 func readAndParseChunk(conv *internal.Conv, r *internal.Reader) ([]byte, []ast.StmtNode, error) {
 	var l [][]byte
+
+	// Regex for ignoring strings of the form /*!50717 SELECT COUNT(*) INTO @rocksdb_has_p_s_session_variables FROM INFORMATION_SCHEMA.TABLES */;
+	// These system generated SQL statements are currently not supported by parser and return error.
```

Comment:
add reference to the bug in the comment

Reply:
added bug reference

Reply:
this check will slow down the parsing, but let's make optimizing the parsing an independent activity